packages-exp/firebase-exp/rollup.config.js

```diff
@@ -121,6 +121,43 @@ const appBuilds = [
   }
 ];
 
+const messagingBuilds = [
+  /**
+   * Messaging Browser Builds
+   */
+  {
+    input: 'messaging/index.ts',
+    output: [
+      {
+        file: resolve('messaging', appPkg.main),
+        format: 'cjs',
+        sourcemap: true
+      },
+      {
+        file: resolve('messaging', appPkg.module),
+        format: 'es',
+        sourcemap: true
+      }
+    ],
+    plugins: [...plugins, typescriptPlugin],
+    external
+  },
+
+  /**
+   * Messaging UMD Builds. Required for FM SDK to work but sw does yet support modules.
+   */
+  {
+    input: 'messaging/index.cdn.ts',
+    output: {
+      file: 'firebase-messaging.js',
+      sourcemap: true,
+      format: 'umd',
+      name: `${GLOBAL_NAME}.app`
```

Comment:
Following the pattern for other components, it seems like it should be `${GLOBAL_NAME}.messaging` here and then there's also a
```
globals: {
      '@firebase/app-exp': `${GLOBAL_NAME}.app`
    },
```

Unless something is different about this case?